nh_eobs_mental_health/views/wardboard_view.xml

```diff
@@ -0,0 +1,18 @@
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="view_mental_wardboard_form">
+            <field name="name">Mental Wardboard Form View</field>
+            <field name="model">nh.clinical.wardboard</field>
+            <field name="inherit_id" ref="nh_eobs.view_wardboard_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//form/header" position="inside">
+                    <button name="toggle_obs_stop" string="Stop Observations" type="object" groups="nh_clinical.group_nhc_ward_manager" attrs="{'invisible':[['obs_stop','=',False]]}"/>
+                    <button name="toggle_obs_stop" string="Restart Observations" type="object" groups="nh_clinical.group_nhc_ward_manager" attrs="{'invisible':[['obs_stop','=',True]]}"/>
```

Comment:
Conditional string does not seem to work. Just changes the string to the expression